docker/nginx/dev.dockerfile

```diff
@@ -3,6 +3,11 @@ FROM nginx:stable-alpine
 ENV TARGET_HOST vor_app
 ENV TARGET_PORT 8080
 
+COPY ./ssl /ssl
```

Comment:
I don't think we should copy the certificate into the container. The Postgres image did so because it was used only for development and I ran into some problems mounting it from docker-compose 😅. 

It might also be easier, in this case, to just mount the certificate when the container run.

Reply:
will update it